gpu/test_gpu.py

```diff
@@ -19,7 +19,7 @@ def verify_instance_gpu_agent(self, name):
 
   def verify_instance_cudnn(self, name):
     self.assert_instance_command(
-        name, "/usr/sbin/ldconfig -p | grep libcudnn" )
+        name, "/sbin/ldconfig -p | grep libcudnn" )
```

Comment:
I think `sudo ldconfig -p` will be more universal?

Reply:
Ack